### PR TITLE
Drop outdated ELB docs.

### PR DESCRIPTION
**Pull request by jmcarp**
At this point, the ELB docs do more harm than good, since almost statement in them is out of date, and since we stopped using ELBs for custom customer domains. I think we should just delete these pages.

### CODE REVIEW

**Review by brittag — APPROVED** [review]
This sounds good to me from a content perspective, but I won't merge yet since probably somebody else should review from a technical perspective.

**Comment by cnelson** [comment]
Fixes https://github.com/18F/cg-site/issues/563

**Pull request merged**
